bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/veneer/BigtableHBaseVeneerSettings.java

```diff
@@ -111,16 +104,31 @@
 import java.util.List;
 import java.util.Objects;
 import java.util.Set;
-import java.util.concurrent.TimeUnit;
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.hbase.util.VersionInfo;
 import org.threeten.bp.Duration;
 
 /** For internal use only - public for technical reasons. */
 @InternalApi("For internal usage only")
 public class BigtableHBaseVeneerSettings extends BigtableHBaseSettings {
+
   private static final String BIGTABLE_BATCH_DATA_HOST_DEFAULT = "batch-bigtable.googleapis.com";
 
+  private static final ClientOperationTimeouts DEFAULT_TIMEOUTS =
+      new ClientOperationTimeouts(
+          /* unaryTimeouts= */ new OperationTimeouts(
+              Optional.absent(),
+              Optional.of(Duration.ofSeconds(20)),
+              Optional.of(Duration.ofMinutes(5))),
+          /* scanTimeouts= */ new OperationTimeouts(
+              Optional.of(Duration.ofMinutes(5)),
+              Optional.of(Duration.ofMinutes(10)),
```

Comment:
I thought we decided on 12 hours?

Reply:
5 mins response timeout & 10 mins attempt & absent op which == 12 hrs